_data/hosting_providers.json

```
@@ -2438,5 +2438,15 @@
     "plan": "",
     "reviewed": "2020.09.20",
     "note": "All domains/sites automatically use Let's Encrypt. Multiple sites share a single certificate using the SAN field"
+  },
+      {
+    "name": "Rad Web Hosting",
+    "link": "https://radwebhosting.com",
+    "category": "full",
```

Since the user has to run an explicit command, this is what we call "partial" support.
```suggestion
    "category": "partial",
```